[clang] ERROR: Cannot find tool gcc

building on a gcc-less setup fails:

```bash
$ make
sh configure
*** ERROR: Cannot find tool gcc
Makefile:17: config.mk: No such file or directory
make: *** [Makefile:55: config.mk] Error 1
```

--- a/configure
+++ b/configure
@@ -58,7 +58,7 @@ check_toolchain()
     local bpftool_version
 
     : ${PKG_CONFIG:=pkg-config}
-    : ${CC=gcc}
+    : ${CC=cc}
     : ${OBJCOPY=objcopy}
     : ${CLANG=clang}
     : ${M4=m4}

